src/dlgProfilePreferences.cpp

```diff
@@ -2093,6 +2119,78 @@ void dlgProfilePreferences::slot_logFileNameFormatChange(const int index)
     label_logFileNameExtension->setVisible(isShown);
 }
 
+void dlgProfilePreferences::slot_configDirOptionChange(const int index)
+{
+    Q_UNUSED(index);
+
+    // Show options to set the config directory if 'Select a
+    // directory...' is selected
+    bool isShown = index == 1;
+    label_setConfigDir->setVisible(isShown);
+    lineEdit_configDir->setVisible(isShown);
+    toolButton_setConfigDir->setVisible(isShown);
+
+    switch (index) {
+    case 0:
+        // Clear mConfigDirPath if 'Default' is selected
+        mConfigDirPath.clear();
+        lineEdit_configDir->setText(QDir::homePath() + "/.config/mudlet");
```

Comment:
:earth_asia: **I18n&I10n** How about:
```
        lineEdit_configDir->setText(QStringLiteral("%1/.config/mudlet").arg(QDir::homePath());
```
to avoid the raw literal string...

Mind you, `getMudletPath(...)` was intended to avoid this sort of path hackery and `getMudletPath(mudlet::mainPath)` will return exactly this value (unless you have fiddled with it)!

Reply:
Updated